src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj

```diff
@@ -5,8 +5,8 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Kusto</PackageId>
-    <Description>Provides developers with libraries for the updated kusto platform under Azure Resource manager to create ane manage clusters. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
-    <Version>0.9.0-preview</Version>
+    <Description>Provides management capabilities for Kusto (Azure Data Explorer) clusters.</Description>
```

Comment:
Please remove the `Descriptions` section and move it to `PackageReleaseNotes` instead.